Add unique id to hero container

briannaorg opened the pull request:
Client is using a third party tracking service that only looks for IDs
on elements

Commit:
Add unique id to hero container

Client is using a third party tracking service that only looks for IDs
on elements

aubreypwd commented on wds-hero-widget.php:
```diff
@@ -229,7 +229,7 @@ function wds_hero( $args = array(), $content = false ) {
 		?>
 
-		<div class="hero <?php echo esc_attr( $args['type'] ); ?>-hero <?php echo esc_attr( $args['class'] ); ?>">
+		<div id="id-<?php echo esc_attr( $args['class'] ); ?>" class="hero <?php echo esc_attr( $args['type'] ); ?>-hero <?php echo esc_attr( $args['class'] ); ?>">
```
Is there any way we can add an arg called `id` and use that? A little more work, but probably the _right_ thing to do rather than use the class over again? What do you think? You could even use `class` as a fallback if no `id` is supplied. I also don't think we need to prefix it with `id-`.

Commit:
Add ID to widget controls

Let the user choose the ID name instead of creating it from the class
name

aubreypwd committed:
Fix spacing.

aubreypwd committed:
Fix spacing.

aubreypwd commented on wds-hero-widget.php:
```diff
@@ -249,8 +252,8 @@ function wds_hero( $args = array(), $content = false ) {
 
-							<div class="slider">
-								<img src="<?php echo $src; ?>" alt="<?php echo $alt; ?>" />
+							<div class="slider" style="background: url(<?php echo $src; ?>) no-repeat; background-size: cover;">
```
Is this change a part of the `id` thing? Wondering what kind of effect this has (I prefer it), but I know I originally built it with `<img>`?

briannaorg replied:
That was an artifact of some testing I was doing locally on gaps below images in mobile. I've rolled that back and replaced it with your original code.

Commit:
Remove background image from slider

Left from testing alternate solutions for gaps left below images on
mobile (actual solution: use images in 3:2 aspect ratio instead of 16:9)

aubreypwd committed:
Add closing tag.
